.github/workflows/build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - 1.56 # MSRV
+          - 1.57 # MSRV
```

Comment:
Did you check for other mentions of the MSRV? We should at least update the `rust-version` in the Cargo metadata.

Reply:
I wasn't sure how much of #1140 we wanted to duplicate. Will add rust-version and the readme comment as well.